docs/UserGuide.adoc

```diff
@@ -454,10 +466,11 @@ Restores the Concierge(TM) application to the state before the previous
 _undoable_ command was executed. +
 Format: `undo`
 
-[NOTE]
-====
-Undoable commands: those commands that modify Concierge(TM)'s content (`add`, `checkin`, `checkout` and `clear`).
-====
+[cols="^.^1,<.^11a", frame=none]
+|===
+|image:icon-important.png[Important info]
+|Undoable commands: those commands that modify Concierge(TM)'s content (`add`, `checkin`, `checkout` and `clear`).
```

Comment:
Shouldn't service command come under this as well? Unless WZ didn't make it undoable

Reply:
Same thing about reassign

Reply:
Added thank you!